app/src/ui/changes/changes-list.tsx

```diff
@@ -127,9 +171,9 @@ export class ChangesList extends React.Component<IChangesListProps, {}> {
         oldPath={file.oldPath}
         include={includeAll}
         key={file.id}
+        onContextMenu={this.onItemContextMenu}
```

Comment:
Not sure the best place to file this, but I'll try here. 

Repro steps:

 1. Select a sequence of files
 2. Right-click on the border between two files

<img width="568" src="https://user-images.githubusercontent.com/359239/38950645-c9073878-430b-11e8-9a97-b78217d63dc4.png">

Expected behaviour: context menu appears
Actual behaviour: nothing happens

I think this is due to the `div` inside the `ChangedFile` component having significant margin that ignores events (i.e. you need to click inside this space to trigger the context menu):

<img width="767" src="https://user-images.githubusercontent.com/359239/38950711-01108602-430c-11e8-87db-e12c8468826b.png">

Reply:
Nice catch @shiftkey. I've created https://github.com/desktop/desktop/issues/4471 to track it but I think we should land this PR as-is rather than keep piling on to it. Let's open a new PR for that work so that we can review it in isolation

Reply:
This problem is from current release version.

Reply:
@crea7or yeah, it's not a regression but it became more evident when you have multiple rows as a hit zone. We've moved it out to #4471 to tackle after this.